cypress/integration/InstancesTest.ts

```diff
@@ -0,0 +1,138 @@
+    beforeEach(() => {
+      cy.visit("/");
+    });
+
+    it("Submit", () => {
+      const newInstanceName: string = "Some new instance";
+      cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
+      cy.ouiaId("create-instance", "PF4/ModalContent")
+        .within(() => {
+          cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+          cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+            "have.text",
+            "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+          );
+          cy.ouiaId("submit", "PF4/Button").click();
+        })
+        .should("not.exist");
+      //TODO: MGDOBR-706 - uncomment code for the verification that the instance was created (currently mocked server does not support it)
+      /*
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId(newInstanceName, "PF4/TableRow")
+        .should("be.visible");
+      */
+    });
+
+    it("Cancel", () => {
+      const newInstanceName: string = "Canceled instance";
+      cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
+      cy.ouiaId("create-instance", "PF4/ModalContent")
+        .within(() => {
+          cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+          cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+            "have.text",
+            "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+          );
+          cy.ouiaId("cancel", "PF4/Button").click();
+        })
+        .should("not.exist");
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId(newInstanceName, "PF4/TableRow")
+        .should("not.exist");
+    });
+  });
+
+  describe("Instance Page - Instance six", () => {
+    beforeEach(() => {
+      cy.visit("/instance/d7e13602-b046-4120-b377-15d61e21c31");
+    });
+
+    it("Header", () => {
+      cy.ouiaId("instance-name", "PF4/Text").should(
+        "have.text",
+        "Instance one"
+      );
+      cy.ouiaId("actions", "PF4/Dropdown")
+        .should("be.visible")
+        .within(() => {
+          cy.ouiaId("actions", "PF4/DropdownToggle").click();
+          cy.ouiaId("details", "PF4/DropdownItem").should("be.visible");
+          cy.ouiaId("delete", "PF4/DropdownItem").should("be.visible");
+        });
+    });
+
+    it("Details", () => {
+      cy.ouiaId("actions", "PF4/Dropdown")
+        .should("be.visible")
+        .within(() => {
+          cy.ouiaId("actions", "PF4/DropdownToggle").click();
+          cy.ouiaId("details", "PF4/DropdownItem").click();
+        });
+      cy.ouiaId("instance-details-panel")
+        .within(() => {
+          cy.ouiaId("instance-details-name", "PF4/Text")
+            .should("have.text", "Instance one")
+            .should("be.visible");
+          cy.ouiaId("instance-details-id")
+            .should("have.text", "3543edaa-1851-4ad7-96be-ebde7d20d717")
```

Comment:
I would expect this ID to be same as on line
```
      cy.visit("/instance/d7e13602-b046-4120-b377-15d61e21c31");
```

DO I miss sth?

Reply:
Yes ... no mocked data. InstanceDetails.ts contains just some dummy data. This test should fail when we start using real mock data.